automated/linux/signaltest/signaltest.sh

```diff
@@ -42,7 +42,7 @@ fi
 
 background_process_start bgcmd --cmd "${BACKGROUND_CMD}"
 
-"${binary}" -D "${DURATION}" -m -p "${PRIORITY}" -t "${THREADS}" \
+"${binary}" -q -D "${DURATION}" -m -p "${PRIORITY}" -t "${THREADS}" \
```

Comment:
I think we should drop this patch at this point. Currently, there is a problem with the reporting when used the quiet argument:

> When the quiet option is used don't print all threads stats. The first
> thread will pause every 16 cycles for 10000us. This will show up in
> the max values for all other threads when the final results are
> printed:
> 
>  # signaltest -q  -D 2s
>  T: 0 ( 4517) P: 0 C:   3121 Min:      5 Act:    8 Avg:    8 Max:      19
>  T: 1 ( 4518) P: 0 C:   3121 Min:      5 Act:10065 Avg:  639 Max:   10073
> 
> We could also remove the sleep but then the system gets fully loaded
> by the test. Furthermore, we would keep the path pretty hot and that's
> not ideal if one wants to test the eratic signal behavior. So only
> consider the first thread for the stats.


https://lore.kernel.org/linux-rt-users/20200901154657.30198-5-dwagner@suse.de/